Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM postgres:14
 
 RUN apt-get update -y \
-	&& apt-get install -y -qq postgresql-plpython3-14 make python3-pip \
+	&& apt-get install -y -qq postgresql-plpython3-14 postgresql-14-pgtap make python3-pip \
```

Comment:
Do we want this Dockerfile to also be for people to try it out? Should we even care to separate the test dependencies?

Reply:
Filed https://github.com/eto-ai/rikai-pg/issues/7